hh-dep-monitoring-service/src/main/java/ru/hh/school/depmonitoring/service/SyncService.java

```diff
@@ -17,29 +19,44 @@
 public class SyncService {
     private final String githubOrganization;
     private final GithubLoader githubLoader;
-    private final GHRepositoryMapper mapper;
+    private final GHRepositoryMapper repositoryMapper;
     private final RepositoryDao repositoryDao;
+    private final DependencyLoader dependencyLoader;
 
-    public SyncService(String githubOrganization, GithubLoader githubLoader, GHRepositoryMapper mapper, RepositoryDao repositoryDao) {
+    public SyncService(
+            String githubOrganization,
+            GithubLoader githubLoader,
+            GHRepositoryMapper repositoryMapper,
+            RepositoryDao repositoryDao,
+            DependencyLoader dependencyLoader
+    ) {
         this.githubOrganization = githubOrganization;
         this.githubLoader = githubLoader;
-        this.mapper = mapper;
+        this.repositoryMapper = repositoryMapper;
         this.repositoryDao = repositoryDao;
+        this.dependencyLoader = dependencyLoader;
     }
 
     @Scheduled(initialDelay = 60*1000L, fixedDelay = 15*60*1000L)
     @Transactional
     public void syncGithub() {
-        List<GHRepositoryDto> repositoryList= githubLoader.getReposOfOrganization(githubOrganization);
+        List<GHRepositoryDto> repositoryList = githubLoader.getReposOfOrganization(githubOrganization);
         for (GHRepositoryDto repositoryDto : repositoryList) {
             var repositoryOptional = repositoryDao.findOne(repositoryDto.getRepositoryId());
+            Repository repository = repositoryMapper.toEntity(repositoryDto);
             if (repositoryOptional.isPresent()) {
                 if (!Objects.equals(repositoryDto.getUpdatedAt(), repositoryOptional.get().getUpdatedAt())) {
-                    repositoryDao.update(mapper.toEntity(repositoryDto));
+                    repositoryDao.update(repository);
                 }
             } else {
-                repositoryDao.create(mapper.toEntity(repositoryDto));
+                repositoryDao.create(repository);
             }
         }
     }
+
+    @Scheduled(initialDelay = 2*60*1000L, fixedDelay = 15*60*1000L)
```

Comment:
добавь все-таки пробелы
```suggestion
    @Scheduled(initialDelay = 2 * 60 * 1000L, fixedDelay = 15 * 60 * 1000L)
```